apps/site/lib/site_web/query_builders/event_date_range.ex

```diff
@@ -8,8 +8,15 @@ defmodule SiteWeb.EventDateRange do
     end
   end
 
+  def build(%{"year" => year}, current_date) do
+    case Date.from_iso8601(year) do
+      {:ok, date} -> for_year(date)
+      {:error, _error} -> for_year(current_date)
+    end
+  end
+
   def build(_month_missing, current_date) do
```

Comment:
Now that this function has been diversified to work for both month and year, should the "_month_missing" be made generic?  (Doesn't quite matter since it isn't read)